terraform/envs/sbox.tfvars

```diff
@@ -11,6 +11,16 @@ ws_rg                         = "oms-automation"
 num_applications              = 20
 vm_size                       = "Standard_F4s_v2"
 dynatrace_tenant              = "yrk32651"
+vm_backup_alert_emails = [
```

Comment:
Can this be changed to use James new module to trigger an alert in Dynatrace instead